index.bs

```diff
@@ -5290,17 +5290,15 @@ defined in this specification are applicable to [=typedefs=].
 
 In a given implementation of a set of [=IDL fragments=],
 an object can be described as being a <dfn id="dfn-platform-object" export>platform object</dfn>, a
-<dfn id="dfn-user-object" export>user object</dfn>, or neither.  There are two kinds of
-object that are considered to be platform objects:
+<dfn id="dfn-user-object" export>user object</dfn>, or neither.
 
-*   objects that implement a non-[=callback interface|callback=] [=interface=];
-*   objects representing an IDL {{DOMException}}.
+Platform objects are objects that implement a non-[=callback interface|callback=] [=interface=].
 
 <dfn id="dfn-legacy-platform-object" export>Legacy platform objects</dfn> are
```

Comment:
Frankly speaking, I've always disliked the name "legacy platform object," particularly because it's difficult to understand why it's considered "legacy

Reply:
That's not really what this PR is about, though. Happy to discuss renaming things in a dedicated issue. :)

Reply:
Yeah I realized that after writing it down.